srv/salt/ceph/stage/deploy/default.sls

```diff
@@ -1,5 +1,32 @@
+{% set master = salt['master.minion']() %}
+
 include:
   - .core
   - ...restart.mon.lax
   - ...restart.mgr.lax
   - ...restart.osd.lax
+
+enable prometheus module:
```

Comment:
Maybe we should encapsulate these steps like we did with 'core'
```
.core
...restart.mon.lax
...restart.mon.lax
...restart.mon.lax
.monitoring_deploy
```

also, if we have the restart logic in place, we should add it similar to the ...restart.$role.lax approach

Reply:
hmm point taken. Unfortunately there is more to the monitoring deployment (stage 2 has the majority of it).